common/views/slices/index.ts

```diff
@@ -11,6 +11,9 @@ export const components = {
   editorialImageGallery: dynamic(() => import('./EditorialImageGallery')),
   embed: dynamic(() => import('./Embed')),
   gifVideo: dynamic(() => import('./GifVideo')),
+  guide_section_heading: dynamic(() => import('./GuideSectionHeading')),
```

Comment:
are we going to go with underscored for future stuff? Maybe makes sense to start going with the grain I guess

Reply:
oh yeah, slice machine UI generates this for you from the label. You can override it manually, but it still won't let you input anything other than underscores, so its either go with it, override in the UI with something like camelCase, or override in the JSON file itself with hyphens - I opted for going with the grain